### PR TITLE
Closes #129

### LINKED ISSUE #129: Add country population from World Bank
**Explain the dataset you want to add and how it would contribute to the Internet Yellow Pages.**
It could be useful to have the countries' population. For example the ranking ihr.country_dependency gives the relative population coverage of eyeball networks per country, but does not indicate the actual population value.  

World Bank seems to be the authoritative data source. 

**Provide the name of the organization providing the data and the url to the dataset**
- Org name: World Bank
- URL: http://api.worldbank.org/v2/country/all/indicator/SP.POP.TOTL?per_page=400&mrv=1&format=json

**If possible describe how you would like to model the dataset in the Yellow Pages**
Since there is only one value from one dataset per country, I suggest adding it as a node property

- What is the label/type of the nodes used by this dataset?
(n: Country)

### PR DESCRIPTION
**Pull request by JustinLoye**
World Bank crawler to get countries' population, closes  issue #129 

## Description

Connect `Country` to an `Estimate` node meaning that a country has an estimated population of `value`.
```
(:Country)-[:POPULATION {value: 123}]->(:Estimate {name: 'World Bank Population Estimate'})
```

## Motivation and Context

It could be useful to have the countries' population. For example the ranking ihr.country_dependency gives the relative population coverage of eyeball networks per country, but does not indicate the actual population value.

## How Has This Been Tested?
I checked the overall number of countries and the values of the most populated ones.

## Screenshots (if appropriate):

10 most populated countries 
top10_pop

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: iyp/crawlers/worldbank/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+## Graph representation
+
+### Country Estimate
+Connect `Country` to an `Estimate` node meaning that a country has an estimated population of `value`.
+```
+(:Country)-[:POPULATION {value: 123}]->(:Estimate {name: 'World Bank Population Estimate'})
+```
+
+## Dependence
+This crawler depends on crawlers setting the country codes.
```
Comment (m-appel) [inline iyp/crawlers/worldbank/README.md:16]: ~setting the country codes~ creating Country nodes.

**Review by m-appel — CHANGES_REQUESTED** [review]
Looks mostly good to me, see comments for nitpicks. And you should add the crawler to `config.json.example`, probably below the Alive-LG crawlers.

File: iyp/crawlers/worldbank/country_pop.py [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+        # Set last time of modification
+        self.reference['reference_time_modification'] = datetime.strptime(content[0]['lastupdated'],
+                                                                          '%Y-%m-%d').replace(tzinfo=timezone.utc)
+
+        # Get countries present in IYP cc to id mapping
+        country_ids = self.iyp.batch_get_nodes_by_single_prop('Country', 'country_code', create=False, all=True)
+
+        # Get countries and population from World Bank
+        lines = set()
+        for entry in json.loads(req.content)[1]:
```
[the remark this answers is not in the document]
Comment (JustinLoye) [reply iyp/crawlers/worldbank/country_pop.py:43]: You are right, that's why I declared `content` in the first place...

File: iyp/crawlers/worldbank/country_pop.py [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+
+            if entry['indicator']['id'] != 'SP.POP.TOTL':
```
[the remark this answers is not in the document]
Comment (JustinLoye) [reply iyp/crawlers/worldbank/country_pop.py:45]: I checked and it does not happen, so I'll remove it

**Review by m-appel — APPROVED** [review]
Thanks, looks good to me.

**Pull request merged**